tests/scripts/configmap_constants.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+#
+# Copyright (c) 2020, 2021 Red Hat, IBM Corporation and others.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+##### script containing constants for configmap yaml tests #####
+#
+
+# get the absolute path of current directory
+CURRENT_DIR="$(dirname "$(realpath "$0")")"
+pushd ${CURRENT_DIR}/../.. >> /dev/null
+
+# Path to the directory containing yaml files
+configmap="${PWD}/manifests/configmaps"
```

Comment:
I don't think we are having any static yamls, do we need this?

Reply:
Its the path to configmap present in autotune manifest ., To copy the same and change it according to the testcase